sesemi/transforms.py

```diff
@@ -62,7 +66,7 @@ def train_transforms(
     resize: int = 256,
     crop_dim: int = 224,
     scale: Tuple[float, float] = (0.2, 1.0),
-    interpolation: int = 3,
+    interpolation: Enum = TF.InterpolationMode.BILINEAR,
```

Comment:
Using a string value here which is then converted into this enum would make it easier to configure this value in the configs as only primitive types are supported when using `instantiate`.